scripts/asciidoc2html.py

```diff
@@ -114,14 +113,14 @@ def _copy_images(self) -> None:
         for filename in ['cheatsheet-big.png', 'cheatsheet-small.png']:
             src = pathlib.Path('doc') / 'img' / filename
             dst = dst_path / filename
-            shutil.copy(src, dst)
+            shutil.copy(str(src), str(dst))
 
     def _build_website_file(self, root: pathlib.Path, filename: str) -> None:
         """Build a single website file."""
         src = root / filename
         assert self._website is not None    # for mypy
         dst = pathlib.Path(self._website)
-        dst = dst / src.parent.relative_to('.') / (src.stem + ".html")
+        dst = src.parent.relative_to('.') / (src.stem + ".html")
```

Comment:
Is this intended? I think you're reverting 64ffce27f9348e14836528c0313d3048669a29c7 from https://github.com/qutebrowser/qutebrowser/pull/5290#pullrequestreview-417805488 here?

Reply:
Of course not. I had used a branch where that change was not made, I think. 😬
I have edited it back.

At least this teaches me the importance of having a closer look at the diff output.  :sweat_smile:

Would it be a good idea to add some rudimentary test for the script as well? Something that at least checks if the files to be created were actuall written out?